pkg/ibmvpc/parser.go

```diff
@@ -613,13 +633,58 @@ func getNACLconfig(rc *datamodel.ResourcesContainerModel,
 	return nil
 }
 
+func getTgwMap(c *datamodel.ResourcesContainerModel) map[string]*datamodel.TransitGateway {
+	tgwIDToTgw := map[string]*datamodel.TransitGateway{}
+	for _, tgw := range c.TransitGatewayList {
```

Comment:
if already iterating `c.TransitGatewayList`, can compute `tgwToSkip` here?

Reply:
no, this map is to skip and not warn about the skipped transit gateway connection again